panic: runtime error: invalid memory address or nil pointer dereference | logging.SetLoggerConfig |

```log
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x52cef6]

goroutine 1 [running]:
github.com/sirupsen/logrus.(*MutexWrap).Lock(...)
        /home/ubuntu/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:52
github.com/sirupsen/logrus.(*Logger).SetFormatter(0x0, 0x5cc260, 0xc0000b20a0)
        /home/ubuntu/go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/logger.go:326 +0x26
github.com/atc0005/elbow/logging.SetLoggerConfig(0xc0000ca000)
        /mnt/t/github/elbow/logging/logging.go:35 +0x787
main.main()
        /mnt/t/github/elbow/main.go:70 +0x198
```

From a quick glance, it appears I never initialized the logging handle on the config object. That's embarrassing.

--- a/config/config.go
+++ b/config/config.go
@@ -92,6 +92,9 @@ func NewConfig(appName, appDescription, appURL, appVersion string) *Config {
 	// Note: The majority of the default settings are supplied via struct tags
 	var config Config
 
+	// Initialize logger "handle" for later use
+	config.Logger = logrus.New()
+
 	config.AppName = appName
 	config.AppDescription = appDescription
 	config.AppURL = appURL

